examples/trials/mnist-annotation/mnist.py

```diff
@@ -180,11 +180,11 @@ def main(params):
     test_acc = 0.0
     with tf.Session() as sess:
         sess.run(tf.global_variables_initializer())
-        """@nni.variable(nni.choice(50, 250, 500), name=batch_num)"""
-        batch_num = params['batch_num']
-        for i in range(batch_num):
-            batch = mnist.train.next_batch(batch_num)
-            """@nni.variable(nni.choice(1, 5), name=dropout_rate)"""
+        """@nni.variable(nni.choice(1, 4, 8, 16, 32), name=batch_size)"""
+        batch_size = params['batch_size']
+        for i in range(params['batch_num']):
+            batch = mnist.train.next_batch(batch_size)
+            """@nni.variable(nni.choice(0.1, 0.5, 1), name=dropout_rate)"""
```

Comment:
Normally dropout rate 1 does not need to be tried, the output should be 0 if dropout function is implemented correctly.

Reply:
sure, how about 0, 0.1, 0.5?

Reply:
Yes, it is fine.